src/openzaak/components/autorisaties/admin.py

```diff
@@ -117,6 +117,17 @@ def delete_model(self, request, obj):
         secrets.delete()
         super().delete_model(request, obj)
 
+    @transaction.atomic
+    def delete_queryset(self, request, queryset):
+        """Given a queryset, delete it from the database."""
+        client_ids = queryset.values_list("client_ids", flat=True)
+        client_ids = [client_id[0] for client_id in client_ids]
```

Comment:
```suggestion
        client_ids = [client_id for client_id  in client_id_field for client_id_field  in client_ids]
```
Change to something like this


This also apparently works. but I've never used it.
```
import itertools
a = [['a','b'], ['c']]
print(list(itertools.chain.from_iterable(a)))
```

Reply:
Thanks for catching it!
I've used `sum`, `itertools.chain` looks also interesting